fcrepo-import-export-driver/src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -80,6 +80,13 @@ public ArgParser() {
         resourceOption.setArgName("resource");
         configOptions.addOption(resourceOption);
 
+        // Source Resource option
+        final Option sourceOption = new Option("s", "source", true, "Source (URI) data was exported from");
```

Comment:
This looks good. It seems that the source command-line argument only applies to importing. I wonder if this argument (and any such others) should include in their Option description that it only has effect on import?